### DISCUSSION TITLE
Main.kt looking for RootView() but mine is named AppRootView!

### DISCUSSION THREAD

**Discussion by davidakoontz** [skiptools/skip#189] [Q&A] [answered]
I renamed my RootView to AppRootView but it looks like it's hardcoded somewhere:

<img width="514" alt="Screenshot 2024-08-14 at 4 43 25 PM" src="https://github.com/user-attachments/assets/465be852-acd6-4929-91d8-dc67b28c824a">
<img width="508" alt="Screenshot 2024-08-14 at 4 44 58 PM" src="https://github.com/user-attachments/assets/a1d00360-486e-4648-8f0e-8bccc9bd9608">

**Comment by davidakoontz** [comment]
OK - seems one CANNOT do this.

**Comment by marcprux** [comment, marked as answer]
The root view created by the `skip init` command is `ModuleNameRootView`, and this is referenced by the `src/main/kotlin/packagename/Main.kt` file. If you were to change the root view name (which we don't recommend), you should just be able to manually update the `Main.kt` shim to reference the new name.
